Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/LightSensorSamplePage.xaml

```diff
@@ -0,0 +1,71 @@
+﻿<Page
+	x:Class="Uno.Gallery.Views.Samples.LightSensorSamplePage"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:local="using:Uno.Gallery"
+	xmlns:samples="using:Uno.Gallery.Views.Samples"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+	xmlns:smtx="using:ShowMeTheXAML"
+	mc:Ignorable="d">
+
+	<Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+		<local:SamplePageLayout>
+
+			<local:SamplePageLayout.FluentTemplate>
```

Comment:
This can now be updated to use the new `DesignAgnosticTemplate` and `SamplePageLayout.IsDesignAgnostic` property  :)

Similar to what we see [here](https://github.com/unoplatform/Uno.Gallery/blob/7677ac022e072e04ecaba492a4dc1b9fb020b737/Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/FileFolderPickerSamplePage.xaml#L11-L13)

Reply:
Done!